docs/api.rst

```diff
@@ -58,9 +58,9 @@ Delegated mode:
 
    class MyDelegate:
        def wormhole_got_code(self, code):
-           print("code: %s" % code)
+	   print("code: %s" % code)
```

Comment:
This reminds me of something

![tabs-spaces-both](https://i.imgur.com/VyMu86F.png)

Reply:
Good to see you around @shapr ❤️